invenio_app_ils/circulation/mail/tasks.py

```diff
@@ -23,17 +37,58 @@ def log_successful_mail(_, data):
     ))
 
 
-def send_ils_mail(factory, prev_loan, loan, trigger, **kwargs):
+def send_ils_mail(prev_loan, loan, trigger, **kwargs):
     """Send loan email message asynchronously and log the result in Celery.
 
     :param factory: Callable message factory to create Message object.
     :param prev_loan: Previous loan.
     :param loan: Updated loan.
     :param trigger: Loan action trigger.
     """
-    msg = factory(prev_loan, loan, trigger, **kwargs)
+    patron = Patron.get_patron(loan["patron_pid"])
+    factory = loan_message_factory()
+    msg = factory(prev_loan, loan, trigger,
+                  recipients=get_recipients([patron.email]), **kwargs)
     current_app.logger.debug("Attempting to send email '{}' to {}...".format(
         msg.subject, ", ".join(msg.recipients)
     ))
     data = msg.__dict__
     send_email.apply_async((data,), link=log_successful_mail.s(data))
+
+
+def send_overdue_mail(loan, profile_url, **kwargs):
+    """Send loan overdue email message async and log the result in Celery.
+
+    :param loan: the overdue loan.
+    """
+    document = Document.get_record_by_pid(loan["document_pid"])
+    patron = Patron.get_patron(loan["patron_pid"])
+    factory = overdue_loan_message_factory()
+    msg = factory(
+        loan,
+        document_title=document["title"]["title"],
+        patron_email=patron.email,
+        days_ago=circulation_overdue_loan_days(loan),
+        profile_url=profile_url,
+        recipients=get_recipients([patron.email]),
+        **kwargs
+    )
+    current_app.logger.debug("Attempting to send email '{}' to {}...".format(
+        msg.subject, ", ".join(msg.recipients)
+    ))
+    data = msg.__dict__
+    send_email.apply_async((data,), link=log_successful_mail.s(data))
+
+
+@shared_task
+def send_overdue_loans_mail_reminder():
+    """Send email message for loans that are overdue every X days."""
+    days = current_app.config["OVERDUE_LOAN_MAIL_INTERVAL"]
+    loan_search = current_circulation.loan_search
+    overdue_loans = loan_search.get_all_overdue_loans().execute()
+    for hit in overdue_loans.hits:
+        loan = hit.to_dict()
+        overdue_days = circulation_overdue_loan_days(loan)
+        if overdue_days % days == 0:
+            profile_url = ''
```

Comment:
You know what? To avoid this, please removing everything about this profile url, it is too complex.
We are going to put the website link in the footer of the email.